src/services/bulkstatus.service.js

```diff
@@ -85,7 +91,7 @@ async function checkBulkStatus(req, res) {
     };
 
     return response;
-  } else if (bulkStatus.status === 'Completed') {
+  } else if (bulkStatus.status === 'Completed' || bulkStatus.exportedFileCount === bulkStatus.totalFileCount) {
```

Comment:
Is this to address an issue where the files are all processed, but that status never gets set to 'Completed'? If so, I would add a comment about why this is necessary.